framework/package-lock.json

```diff
@@ -10903,6 +10942,118 @@
 			"resolved": "https://registry.npmjs.org/swagger-schema-official/-/swagger-schema-official-2.0.0-bab6bed.tgz",
 			"integrity": "sha1-cAcEaNbSl3ylI3suUZyn0Gouo/0="
 		},
+		"swagger-stats": {
+			"version": "0.95.11",
+			"resolved": "https://registry.npmjs.org/swagger-stats/-/swagger-stats-0.95.11.tgz",
+			"integrity": "sha512-RpE6NEofecAURwUm8Qfh2/IOkaselhHS/qQdTsowPuNrjSk7jkNF+JCqQLIM4xhIFX0i43sARDEmopi0cXLevA==",
+			"requires": {
+				"basic-auth": "^2.0.0",
+				"cookies": "^0.7.1",
+				"debug": "^3.1.0",
+				"moment": "^2.19.3",
+				"natives": "^1.1.6",
```

Comment:
https://www.npmjs.com/package/natives

Reply:
This dep is deprecated: `This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.`